scripts/create-release-archive.js

```diff
@@ -0,0 +1,193 @@
+const child_process = require('child_process') // eslint-disable-line camelcase
```

Comment:
Any reason we don't use `#!/usr/bin/env node` so this can be run as a shell script?

Reply:
Sure, let's do that